### DISCUSSION TITLE
`bun .` doesn't respect `exports["."]`

### DISCUSSION THREAD

**Discussion by vjpr** [oven-sh/bun#3970] [General]
I always expect `bun .` to use my `package.json#exports`.

```
  "exports": {
    ".": {
      "bun": "./src/index.ts",
      "types": "./src/index.ts"
    }
  },
```

```
error: Cannot find module "/xxx/app" from "bun:main"
```

If this is not the desired behaviour of `bun .`, what is the easiest way to run `package.json#exports["."]` from `bun` cli.

**Comment by Jarred-Sumner** [comment]
This is a bug caused by converting the import into an absolute path instead of a package path 

Converting to an issue

https://github.com/oven-sh/bun/issues/3988
